src/writer/proposal.ts

```diff
@@ -65,19 +65,39 @@ export async function verify(body): Promise<any> {
   const isAuthorized = members.includes(body.address.toLowerCase());
 
   if (onlyAuthors && !isAuthorized) return Promise.reject('only space authors can propose');
-
   if (!isAuthorized) {
     try {
       const validationName = space.validation?.name || 'basic';
       const validationParams = space.validation?.params || {};
-      const isValid = await snapshot.utils.validations[validationName](
-        body.address,
-        space,
-        msg.payload,
-        validationParams
-      );
+      const minScore = space.validation?.params?.minScore || space.filters.minScore;
```

Comment:
Filters may not exist here
```suggestion
      const minScore = space.validation?.params?.minScore || space.filters?.minScore;
```